### PR TITLE
Added basic landing page at root URL

### PR DESCRIPTION
**Pull request by shiv-io**
https://github.com/andreagrandi/covid-api/issues/22

Added a plain HTML landing page with "Fork me on GitHub" ribbon and link to documentation.

### CODE REVIEW

**Comment by MatMoore** [comment]
Thanks for working on this @shiv-io !

I'm having some trouble getting it to run though. The first problem seems to be a missing dependency. I got the following stack trace

```
Traceback (most recent call last):
  File "/Users/mat/.pyenv/versions/3.8.1/lib/python3.8/multiprocessing/process.py", line 315, in _bootstrap
    self.run()
  File "/Users/mat/.pyenv/versions/3.8.1/lib/python3.8/multiprocessing/process.py", line 108, in run
    self._target(*self._args, **self._kwargs)
  File "/Users/mat/code/covid-api/env/lib/python3.8/site-packages/uvicorn/subprocess.py", line 61, in subprocess_started
    target(sockets=sockets)
  File "/Users/mat/code/covid-api/env/lib/python3.8/site-packages/uvicorn/main.py", line 382, in run
    loop.run_until_complete(self.serve(sockets=sockets))
  File "uvloop/loop.pyx", line 1456, in uvloop.loop.Loop.run_until_complete
  File "/Users/mat/code/covid-api/env/lib/python3.8/site-packages/uvicorn/main.py", line 389, in serve
    config.load()
  File "/Users/mat/code/covid-api/env/lib/python3.8/site-packages/uvicorn/config.py", line 288, in load
    self.loaded_app = import_from_string(self.app)
  File "/Users/mat/code/covid-api/env/lib/python3.8/site-packages/uvicorn/importer.py", line 23, in import_from_string
    raise exc from None
  File "/Users/mat/code/covid-api/env/lib/python3.8/site-packages/uvicorn/importer.py", line 20, in import_from_string
    module = importlib.import_module(module_str)
  File "/Users/mat/.pyenv/versions/3.8.1/lib/python3.8/importlib/__init__.py", line 127, in import_module
    return _bootstrap._gcd_import(name[level:], package, level)
  File "<frozen importlib._bootstrap>", line 1014, in _gcd_import
  File "<frozen importlib._bootstrap>", line 991, in _find_and_load
  File "<frozen importlib._bootstrap>", line 975, in _find_and_load_unlocked
  File "<frozen importlib._bootstrap>", line 671, in _load_unlocked
  File "<frozen importlib._bootstrap_external>", line 783, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "./covidapi/app.py", line 3, in <module>
    from starlette.staticfiles import StaticFiles
  File "/Users/mat/code/covid-api/env/lib/python3.8/site-packages/starlette/staticfiles.py", line 7, in <module>
    from aiofiles.os import stat as aio_stat
ModuleNotFoundError: No module named 'aiofiles'
```

I can fix this by adding `aiofiles` to `requirements.txt` though.

The other error is due to missing files:

```
Traceback (most recent call last):
  File "/Users/mat/.pyenv/versions/3.8.1/lib/python3.8/multiprocessing/process.py", line 315, in _bootstrap
    self.run()
  File "/Users/mat/.pyenv/versions/3.8.1/lib/python3.8/multiprocessing/process.py", line 108, in run
    self._target(*self._args, **self._kwargs)
  File "/Users/mat/code/covid-api/env/lib/python3.8/site-packages/uvicorn/subprocess.py", line 61, in subprocess_started
    target(sockets=sockets)
  File "/Users/mat/code/covid-api/env/lib/python3.8/site-packages/uvicorn/main.py", line 382, in run
    loop.run_until_complete(self.serve(sockets=sockets))
  File "uvloop/loop.pyx", line 1456, in uvloop.loop.Loop.run_until_complete
  File "/Users/mat/code/covid-api/env/lib/python3.8/site-packages/uvicorn/main.py", line 389, in serve
    config.load()
  File "/Users/mat/code/covid-api/env/lib/python3.8/site-packages/uvicorn/config.py", line 288, in load
    self.loaded_app = import_from_string(self.app)
  File "/Users/mat/code/covid-api/env/lib/python3.8/site-packages/uvicorn/importer.py", line 20, in import_from_string
    module = importlib.import_module(module_str)
  File "/Users/mat/.pyenv/versions/3.8.1/lib/python3.8/importlib/__init__.py", line 127, in import_module
    return _bootstrap._gcd_import(name[level:], package, level)
  File "<frozen importlib._bootstrap>", line 1014, in _gcd_import
  File "<frozen importlib._bootstrap>", line 991, in _find_and_load
  File "<frozen importlib._bootstrap>", line 975, in _find_and_load_unlocked
  File "<frozen importlib._bootstrap>", line 671, in _load_unlocked
  File "<frozen importlib._bootstrap_external>", line 783, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "./covidapi/app.py", line 11, in <module>
    app.mount("/index.html", StaticFiles(directory=static_dir), name="static")
  File "/Users/mat/code/covid-api/env/lib/python3.8/site-packages/starlette/staticfiles.py", line 55, in __init__
    raise RuntimeError(f"Directory '{directory}' does not exist")
RuntimeError: Directory '/Users/mat/code/covid-api/covidapi/api/static' does not exist
```

I think the `index.html` is missing from the commit?

File: covidapi/api/views.py [hunk 1]
```diff
@@ -8,6 +11,9 @@
 
 router = APIRouter()
 
+@router.get('/')
+def root():
```
Comment (MatMoore) [inline covidapi/api/views.py:14]: Do you need both this and the `app.mount(..., StaticFiles(...))` line below? It seems like you could do the same thing with only the `app.mount` call.

Comment (shiv-io) [reply covidapi/api/views.py:14]: Noted! I went ahead and got rid off the `app.mount` call and left the base route as-is.

**Comment by shiv-io** [comment]
@MatMoore I've incorporated the changes to the latest commit.

File: covidapi/app.py [hunk 2]
```diff
@@ -2,4 +2,4 @@
 from .api import views
 
 app = FastAPI()
-app.include_router(views.router)
+app.include_router(views.router)
```
Comment (andreagrandi) [inline covidapi/app.py]: Missing new line at the end of file

File: covidapi/api/static/index.html [hunk 3]
```diff
@@ -0,0 +1,14 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <title>COVID-19 Python API</title>
```
Comment (MatMoore) [inline covidapi/api/static/index.html:5]: We could add some CSS here to make this look nicer but I think it would be better to do that as another PR. Does that make sense to you @andreagrandi?

**Review by MatMoore — APPROVED** [review]

File: covidapi/api/static/index.html [hunk 4]
```diff
@@ -0,0 +1,14 @@
...
+</head>
+<body>
+<a href="https://github.com/andreagrandi/covid-api"><img width="149" height="149" src="https://github.blog/wp-content/uploads/2008/12/forkme_left_red_aa0000.png?resize=149%2C149" class="attachment-full size-full" alt="Fork project on GitHub" data-recalc-dims="1"></a>
```
Comment (MatMoore) [inline covidapi/api/static/index.html]: These image tags seem to come with some existing attributes and classes which seems a bit strange if there's no CSS/Javascript that references them - I'm assuming it looks the same if we strip these out?

**Review by MatMoore — APPROVED** [review]

File: covidapi/api/static/index.html (hunk 4 above)
Comment (MatMoore) [reply covidapi/api/static/index.html]: These attributes:
- `class="attachment-full size-full"` (and `class="middle"` below)
-  `data-recalc-dims="1"`

File: covidapi/api/static/index.html (hunk 3 above)
Comment (andreagrandi) [reply covidapi/api/static/index.html:5]: @MatMoore yes, it makes sense to do that in a different PR. I wonder if there are any free templates we can borrow from, so we don't have to invest to much time into any design.

File: covidapi/api/static/index.html (hunk 4 above)
Comment (shiv-io) [reply covidapi/api/static/index.html]: Resolved: https://github.com/andreagrandi/covid-api/pull/23/commits/a08df94b13749f888227e04a77bb1e2b3085b148
